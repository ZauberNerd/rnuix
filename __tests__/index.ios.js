import React from 'react';
import { shallow } from 'enzyme';
import App from '../index.ios.js';

it('renders correctly', () => {
    const tree = shallow(<App />);
    expect(tree).toMatchSnapshot();
});
