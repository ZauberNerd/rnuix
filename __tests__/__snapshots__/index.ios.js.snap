// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": [Function],
  "nodes": Array [
    [Function],
  ],
  "options": Object {},
  "renderer": Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
